tollbros/tollbrothers-ui
# Charles/horizontal scroller

## JIRA Ticket
https://tbnextjs.atlassian.net/browse/NJP-1570

## Describe Changes in this Pull Request
- Custom scroll bar
- Handles variable slide sizes
- Extends to edge of screen
- Optional next and previous arrows
- When sliding left or right, slider sticks to center image

## Testing Instructions

- Pull https://github.com/tollbros/tollbrothers-ui/tree/charles/horizontalScroller from the `tollbrothers-ui` repo
- Pull https://bitbucket.org/tollbrothers/com.tollbrothers.www/branch/charles/NJP-2144-add-horizontal-scroller-from-ui from the `tollbrothers.com` repo
- Update `package.json` with `"@tollbrothers/tollbrothers-ui": "file:../tollbrothers-ui"` if it isn't already
- Run `npm i`
- Run `npm run dev`
- http://localhost:3000/luxury-homes-for-sale/Arizona/Sterling-Grove#gallery is a good place to start testing
- `http://localhost:3000/luxury-homes-for-sale/North-Carolina/Forestville-Village-by-Toll-Brothers/Hemlock-Collection/Barlow#gallery` also check under _Unique exteriors that set your home apart_



### Tasks:

1. [ ] Changes Meet JIRA Tickets Requirements
2. [ ] Commit message with `fix` or `feat` to trigger new release
3. [ ] Tested changes on Mobile, Tablet and Desktop screen sizes
4. [ ] Tested changes on Chrome, FireFox and Safari
5. [ ] Demoed to Design Team (For new features only)
6. [ ] Notified Digital Marketing Team if changes will impact A/B Tests
7. [ ] Checked Tracking/Analytics (Notified Digital Marketing Team if changes impact tracking)
